Add python mako dependency into the Dockerfile

My dockerized lightning-based project started failing after 6c240ab589388a6319ff5736bee35944120c681e.

I fixed it on my side with https://github.com/darwin/simverse/commit/13d2122edd9d4a04dcab7e75b4105e6a68a2ad01. You should do the same.

As a bonus. You could also consider adding a test build of your Dockerfile into CI.

Repro as of bb301040e43a5ba4cb6b1423b4869183147b6655:
```
docker build .
```
```
...
tools/generate-wire.py hsmd/gen_hsm_wire.h hsm_wire_type < hsmd/hsm_wire.csv > hsmd/gen_hsm_wire.c
tools/generate-wire.py common/gen_status_wire.h status < common/status_wire.csv > common/gen_status_wire.c
tools/generate-bolts.py --page impl wire/gen_peer_wire.h wire_type < wire/gen_peer_wire_csv > wire/gen_peer_wire.c
tools/generate-wire.py gossipd/gen_gossip_wire.h gossip_wire_type < gossipd/gossip_wire.csv > gossipd/gen_gossip_wire.c
Traceback (most recent call last):
  File "tools/generate-bolts.py", line 23, in <module>
    from mako.template import Template
ModuleNotFoundError: No module named 'mako'
tools/generate-wire.py --header gossipd/gen_gossip_store.h gossip_store_type < gossipd/gossip_store.csv > gossipd/gen_gossip_store.h
make: *** [wire/Makefile:71: wire/gen_peer_wire.c] Error 1
make: *** Waiting for unfinished jobs....
The command '/bin/sh -c ./configure --prefix=/tmp/lightning_install && make -j3 DEVELOPER=${DEVELOPER} && make install' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,6 +11,7 @@ RUN apk add --no-cache \
      python \
      python-dev \
      python3 \
+     python3-mako \
      sqlite-dev \
      wget \
      git \

